### ISSUE TITLE
UI suggestions

### ISSUE THREAD

**Issue by x11x** [ReDEnergy/SessionSync#45]
Hi,
I have some minor UI suggestions. Maybe these could be implemented as options if you did not want them enabled by default.
 - Double-click instead of single-click on a tab to open the URL. I think single-click should either select the tab (once #9 is done), show extra info or context for the item, or do nothing.
 - Opening URLs in a new tab. Maybe support Ctrl-Click (preferably Ctrl-Double Click) if you did not want this as the default behaviour.
 - (I will repeat this here, I also mentioned it in #32) Action buttons on hover for items (... menu, or some commonly used actions like delete, rename, etc.)
 - Labels next to buttons for the commonly used items like the Save Session heart button (I read this suggestion on one of the other issues too, I would like to second it). I like the tool tips, they are easy to use, but seeing as there is space, you could have labels on the commonly used functions and still omit them from the other functions. Also, what about an Android style ... menu to keep all the other functions uncluttered?)
Again, love the extension, thanks.

**Comment by ReDEnergy** [comment]
- Double click to open - perfect and valid idea, I will add an option
- Opening URLs in a new tab
  - Do you know that you can use middle-click for that
  - Also you can switch between these actions (click and middle-click) from the settings
- Action buttons on hover for items - Definitely will
- Labels and the rest of the suggestions - yes actually I was really planing to add this small menu button where you can configure what buttons appear and if labels should be visible or not (basically will act exactly like the Firefox overflow menu for addons. All buttons that are not on that bar will be sent) So your most used one will always be visible while you can hide the ones that you never or rarely use.

Will try to bring as many of these feature request as possible in the next release (seems doable).

**Comment by rayman89** [comment]
Yes the heart for saving current session was always strange to me. A diskette or a down pointing arrow would be more intuitive IMO.

**Comment by ReDEnergy** [comment]
Nice suggestion. I'll look for some new icons, can't say that I like them that much but all of them are from a great free pack and as there are not that many complete packs out there (and also free) It was one of the best options. I'll try to look for some new packs. Maybe even offer multiple icon themes options.
